build/pipelines/templates/extensions-ci.yml

```diff
@@ -0,0 +1,106 @@
+parameters:
+- name: ExtensionDirectory
+  type: string
+  default: not-specified
+
+jobs:
+- job: "Build_And_Test_Windows"
+  pool:
+    vmImage: 'windows-latest'
+
+  variables:
+    - template: extensions-variables.yml
+
+  steps:
+  
+  - template: ../../install-dotnet.yml
+  - task: DotNetCoreCLI@2
+    displayName: 'Build projects'
+    inputs:
+      command: 'build'
+      arguments: '-c Release -p:BuildNumber=$(buildNumber) -p:IsLocalBuild=False'
+      projects: ${{ parameters.ExtensionDirectory }}/src/*.csproj
+
+  #  Extensions test structure must be defined
+  # - task: DotNetCoreCLI@2
```

Comment:
nit: do we need to keep this in, are we expecting to uncomment any time soon? could we use source control to find this snippet later if we need it and delete the comment?

Reply:
I can remove, but I expect to follow up here soon for the tests. We need to move some of the tests but that is really outside of the scope of this PR